package.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
@abhinavpreetu should we commit `yarn.lock`? what does other postcss plugins do?

Reply:
Some do, some do not. But ideally we should commit the lock file, right?